packages/driver/src/dom/selection.js

```diff
@@ -362,6 +362,35 @@ const _moveCursorUpOrDown = function (el, up) {
   }
 }
 
+const moveCursorToStart = (el) => {
+  return _moveCursorToStartOrEnd(el, true)
+}
+
+const moveCursorToEnd = (el) => {
+  return _moveCursorToStartOrEnd(el, false)
+}
+
+const _moveCursorToStartOrEnd = function (el, start) {
+  if ($elements.isTextarea(el) || $elements.isInput(el)) {
+    const { start, end } = getSelectionBounds(el)
+    const returnValue = start ? 0 : $elements.getNativeProp(el, 'value').length
+
+    if (start !== end) {
+      return _collapseInputOrTextArea(el, returnValue)
+    }
+
+    //# Don't worry about moving past the end of the string
+    //# nothing will happen and there is no error.
+    return setSelectionRange(el, returnValue, returnValue)
+  }
+
+  if ($elements.isContentEditable(el)) {
+    const selection = _getSelectionByEl(el)
+
+    return $elements.callNativeMethod(selection, 'modify', 'move', start ? 'backward' : 'forward', 'paragraphboundary')
+  }
+}
+
```

Comment:
this is the code I'm asking about being pulled from #2436 , it's obviously not in develop yet if it's showing up as new in this PR

Reply:
oh ok, got it. no, I wrote it myself using MDN documentation [here](https://developer.mozilla.org/en-US/docs/Web/API/Selection/modify)

Reply:
oh, I was confused by the `//#` in comments which happens when old coffeescript code is converted to js. I guess you just thought that was a cypress convention :laughing:

Reply:
oh, sorry about that, will remove the comment tomorrow, when I will also integrate your other feedback related to the scroll. thanks very much!